src/v/config/config_store.h

```diff
@@ -87,10 +87,10 @@ class config_store {
                         errors[name] = fmt::format(
                           "Validation error: {}",
                           validation_err.value().error_message());
+                    } else {
+                        found->second->set_value(node.second);
+                        ok = true;
```

Comment:
nice. I wonder if this covers that scenario we found in previous iterations of debug-bundle project where validation on some configs just did not trigger?

Reply:
If the config was in the `.bootstrap.yaml` file then yeah maybe!